.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: nix
 
-env:
-  NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/c0b1e8a5fb174cd405dcca9f7fec275714ad9f4b.tar.gz
-
 install:
+  - source scripts/set_nixpath.sh
   - nix-env -iA nixopsUnstable -f '<nixpkgs>'
```

Comment:
One particular idea (from https://github.com/input-output-hk/iohk-nixops/blob/devops-123-refactor/shell.nix#L45) is to:

1. use `shellHook` in a `shell.nix` to set `NIX_PATH`.
2. standartise on _not_ running anything outside of a nix shell

While it seems uncontroversial to do 1 _fully_, the extent to which we go with 2 is arguably open to debate.

I don't want to sound judgemental here -- but merely suggest this for discussion.

..and I think, that in today's standup meeting, when I brought this up (although the _nothing outside nix-shell_ wasn't worded strongly) Domen was in agreement.

What do you think?

cc @domenkozar